test/test_readability.py

```diff
@@ -91,27 +91,35 @@ def test_count_sentences_words_syllables():
 
 def test_flesch_score_range():
     for text in text_dict:
-        assert readability.Flesch().get_score_from_text(text) > 0
-        assert readability.Flesch().get_score_from_text(text) < 100
+        assert readability.FleschKincaidScorer().get_score_from_text(text) > 0
+        assert readability.FleschKincaidScorer().get_score_from_text(text) < 100
 
-    assert readability.Flesch().get_score_from_text(easy_text1) > readability.Flesch().get_score_from_text(
+    assert readability.FleschKincaidScorer().get_score_from_text(
+        easy_text1
+    ) > readability.FleschKincaidScorer().get_score_from_text(
         medium_text1
     ), "Expected easy text to be easier than medium text"
-    assert readability.Flesch().get_score_from_text(medium_text1) > readability.Flesch().get_score_from_text(
+    assert readability.FleschKincaidScorer().get_score_from_text(
+        medium_text1
+    ) > readability.FleschKincaidScorer().get_score_from_text(
         medium_hard_text
     ), "Expected medium text to be easier than medium hard text"
-    assert readability.Flesch().get_score_from_text(medium_text2) > readability.Flesch().get_score_from_text(
+    assert readability.FleschKincaidScorer().get_score_from_text(
+        medium_text2
+    ) > readability.FleschKincaidScorer().get_score_from_text(
         medium_hard_text
     ), "Expected medium text to be easier than medium hard text"
-    assert readability.Flesch().get_score_from_text(medium_hard_text) > readability.Flesch().get_score_from_text(
+    assert readability.FleschKincaidScorer().get_score_from_text(
+        medium_hard_text
+    ) > readability.FleschKincaidScorer().get_score_from_text(
         hard_text1
     ), "Expected medium hard text to be easier than hard text"
 
 
 def test_flesch_score_against_estimate():
     for text in text_dict:
-        flesch_score = readability.Flesch().get_score_from_text(text)
-        feedback = readability.Flesch().get_feedback(flesch_score)
+        flesch_score = readability.FleschKincaidScorer().get_score_from_text(text)
+        feedback = readability.FleschKincaidScorer().get_feedback(flesch_score)
```

Comment:
For instance, shouldn't get_feedback() be a static method? Looks like a pure function.

Reply:
You are right. Fixed